diagnostic.md

```diff
@@ -33,6 +36,14 @@ methods.
 
 <!-- your answer starts here -->
 
+getters -
+$( '#header' );
+$( 'li' );
+
+setters -
+$( document.body.children[0] );
+$( [ window, document ] );
+
```

Comment:
For example and getter would be `$('header').text()` and a setter would be `$('header').text('text you want to set')`